docs/monitoring.md

```diff
@@ -341,6 +341,16 @@ Security options for the Spark History Server are covered more detail in the
     </td>
     <td>2.3.0</td>
   </tr>
+  <tr>
+    <td>spark.history.store.serializer</td>
+    <td>JSON</td>
+    <td>
+        Serializer for writing/reading in-memory UI objects to/from disk-based KV Store; JSON or PROTOBUF.
+        JSON serializer is the only choice before Spark 3.4.0, thus it is the default value.
+        PROTOBUF serializer is fast and compact, and it is the default serializer for disk-based KV store of live UI.
```

Comment:
Users cannot leverage/access the live db right ? (Unlike history db, which is used across restarts).
Do we want to expose the impl details ? If we change it tomorrow to something else, it should be completely transparent to users.

Reply:
ok, updated